### ISSUE TITLE
Volume list error: template: OutputYamlTpl:24:74

### ISSUE THREAD

**Issue by deepak71** [openebs/maya#870]
## Is this a BUG REPORT or FEATURE REQUEST?
 
 Choose one: BUG REPORT

**What happened**:
When executing mayactl volume list, received When executing mayactl volume list, received "Volume list error: template: OutputYamlTpl:24:74: executing "OutputYamlTpl" at <"running">: invalid value; expected string"

**What you expected to happen**:
Receive a volume list

**How to reproduce it (as minimally and precisely as possible)**:

- connect to maya-api-server container in kubernetes
- execute mayactl volume list

**Environment**:
- Maya version (use `maya version`): Version: 0.8.0-unreleased
- M-apiserver version (use `m-apiserver version`): 
- OS (e.g. from /etc/os-release): "Container Linux by CoreOS 1911.4.0 (Rhyolite)"
- Kernel (e.g. `uname -a`):  Linux 4.14.81-coreos
- Install tools:
- Others: K8s worker node

**Comment by ashishranjan738** [comment]
Seems this bug is fixed in 0.8 release. Install it from here: https://github.com/openebs/openebs/releases

**Comment by davidq2q** [comment]
Issue still occurs on 0.8.1 release through helm.
```/ # mayactl volume list
Volume list error: template: OutputYamlTpl:27:74: executing "OutputYamlTpl" at <"running">: invalid value; expected string```
